src/test/java/bio/terra/service/resourcemanagement/OneProjectPerResourceUnitTest.java

```diff
@@ -109,20 +118,6 @@ public void teardown() {
     resourceConfiguration.setDataProjectPrefix(dataProjectPrefix);
   }
 
-  @Test
-  public void snapshotDatasetSeparateProjects() throws Exception {
-    Dataset dataset = createDataset(billingProfiles.get(0), projects.get(0));
-    String datasetProjectId = projects.get(0).getGoogleProjectId();
-    datasets.add(dataset);
-
-    // this will now always be random and different than the dataset project id
-    String snapshotProjectId = oneProjectPerResourceSelector.projectIdForSnapshot();
-    assertThat(
-        "Project ID for snapshot is not the same as the dataset's project id",
-        snapshotProjectId,
-        not(datasetProjectId));
-  }
```

Comment:
I wonder if we could replicate this test by actually creating a dataset & snapshot and then confirming that they live in two different projects.  (Probably a follow-on test addition - I'd definitely like to increase our tests now that we actually create projects in the tests!)

Reply:
I've added tests in `src/test/java/bio/terra/service/resourcemanagement/RBSConnectedTest.java`